modules/islandora_crossref_citations/README.md

```diff
@@ -0,0 +1,49 @@
+# Islandora Crossref Citations
+
+## Introduction
+
+Islandora Crossref Citations generates a block which displays citation counts for any object with a DOI, via the Crossref Cited-by service, for use with Islandora Badges.
+
+The badge will only display on objects with a MODS datastream and a DOI (Digital Object Identifier).
+
+## Requirements
+
+This module requires the following modules/libraries:
+
+* [Islandora](https://github.com/islandora/islandora)
+* [Islandora Scholar](https://github.com/Islandora/islandora_scholar)
+* [Islandora Badges](../../)
+
+## Installation
+
+Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
+
+## Configuration
+
+Configuration path is admin/islandora/tools/badges/crossref (Administration > Islandora > Islandora Utility Modules > Islandora Badges Configuration > Crossref citations).
+
+Block text: Defines the text used in the citation count block. Defaults to "Citations via Crossref".
+
+Badge type: You can choose between plain text or an automatically-generated image. Plain text is offered for custom styling.
```

Comment:
How is  badge image generating website used? io? could  that be added in the readme?